content/en/APIUseCases/create_asset.md

```diff
@@ -0,0 +1,229 @@
+https://docs.cobalt.io/v2/#errors?  -->
+
+## Find Your Asset ID
+
+To add or modify or information related to your asset, you'll need the asset ID.
+You can find this ID with the REST call to [Get All Assets](https://docs.cobalt.io/v2/#get-all-assets):
+
+```
+curl -X GET "https://api.cobalt.io/assets" \
+  -H "Accept: application/vnd.cobalt.v2+json" \
+  -H "Authorization: Bearer YOUR-PERSONAL-API-TOKEN" \
+  -H "X-Org-Token: YOUR-V2-ORGANIZATION-TOKEN" \
+  | jq .
+```
+
+If you've set up more than one Asset, you may need to search through the output.
+For more information about each Asset response field, see our API documentation
+reference to [Get All Assets](https://docs.cobalt.io/v2/#get-all-assets).
+
+{{% expand "Review sample output." %}}
+```
+{
+  "data": {
+    "resource": {
+      "id": "YOUR-ASSET-IDENTIFIER",
+      "title": "Test Asset",
+      "description": "How to describe the asset to our pentesters",
+      "asset_type": "web",
+      "logo": null,
+      "attachments": []
+    },
+    "links": {
+      "ui": {
+        "url": "https://api.cobalt.io/links/<endpoint for the asset>"
+      }
+    }
+  }
+}
+```
+{{% /expand %}}
+
+If you've set up more than one asset, you'll see the `id` in the same
+code block as the `title`, which you may have used to [create the asset](#create-an-asset).
+
+Save the value of the asset `id` as `YOUR-ASSET-IDENTIFIER`. You'll use that ID,
+which starts with `as_`, when updating or uploading information to your asset.
+
+## Add or Modify Asset Details
+
+Now that you've created an asset and have the asset ID, you can add more
+information with the following REST call:
+
+
+```
+curl -X PUT 'https://api.cobalt.io/assets/YOUR-ASSET-IDENTIFIER' \
+  -H 'Accept: application/vnd.cobalt.v2+json' \
+  -H 'Authorization: Bearer YOUR-PERSONAL-API-TOKEN' \
+  -H 'Content-Type: application/vnd.cobalt.v2+json' \
+  -H 'X-Org-Token: YOUR-V2-ORGANIZATION-TOKEN' \
+  --data '{
+            "title": "Updated title",
+            "description": "Updated description",
+            "asset_type": "web",
+            "size": "m",
+            "coverage": "standard"
+          }' \
+  -v
+```
+
+You can find optional values for `size` and `coverage` in our API reference on
+how to [Update an Asset](https://docs.cobalt.io/v2/#update-an-asset).
+
+When you review the output of the REST call with the `-v`, look for the line
+with `HTTP/2`. You'll see one of the following lines:
+<!-- The output is associated with a `201` message, which doesn't include
+results, which is why I recommend a `-v` -->
+
+| Message    | Meaning          |
+|------------|------------------|
+| HTTP/2 201 | Asset created    |
+| HTTP/2 401 | No asset created. Check the value of your API Token|
+| HTTP/2 404 | No asset created. Check the value of YOUR-V2-ORGANIZATION-TOKEN|
+| HTTP/2 409 | No asset created |
+
+
+## Include an Asset Attachment
+
+You can help our pentesters by including images or even PDFs. As noted in our
```

Comment:
PDF files

Reply:
Maybe it's time to set up another variable, to list the same attachments on this page, and in https://developer.cobalt.io/getting-started/assets/asset-description/